layers/meta-resin-beaglebone/recipes-core/fix-mmc-bbb/fix-mmc-bbb.bb

```diff
@@ -0,0 +1,11 @@
+DESCRIPTION = "Apply memory pressure fix"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
```

Comment:
You don't deploy any files in this directory so I suspect it being useless - unless there is something I miss.